tests/integration/memcached/test_add_servers.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should report instance metrics when multiple servers are
+added at once
+*/
+
+/*SKIPIF
+<?php require('skipif.inc'); ?>
+*/
+
+/*INI
+*/
+
+/*EXPECT_METRICS_EXIST
+Datastore/instance/Memcached/host1/1
+Datastore/instance/Memcached/host2/2
+Datastore/instance/Memcached/host3/11211
+*/
+
+require_once(realpath (dirname ( __FILE__ )) . '/../../include/helpers.php');
+require_once(realpath (dirname ( __FILE__ )) . '/../../include/tap.php');
+require_once(realpath (dirname ( __FILE__ )) . '/memcache.inc');
+
+$memcached = new Memcached();
+$memcached->addServers(array(
```

Comment:
Can you add a test case for the `0` port option that specifies a unix socket/localhost?

Reply:
added in [8a3ad31](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/8a3ad31b2cec45c97f685b7bf191cfec56cea406)

Reply:
Can we also add a test where an empty array is passed to `addServers()`?

Reply:
added in [3cef8c3](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/3cef8c34afaa4358f2fa080e2d865b2c84390f76)